index.html

```diff
@@ -31,7 +31,139 @@ <h1>I’m Mariane
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
+   <section>
+    <article class="snapshoot"><img src="images/Snapshoot3.png" alt=""></article>
+    <article>
+     <h1>Multi-Post Stories</h1>
+     <div class="frame">
+      <h2 class="tc">CANOPY</h2>
+      <div></div>
+      <p >Back End Dev</p>
+      <div></div>
+      <p>2015</p>
+     </div>
+     <div class="text">
+      <p>A daily selection 
+       of privately personalized reads; no accounts or sign-ups required.</p>
+     </div>
+     <div class="lang">
+      <p>html</p>
+      <p>css</p>
+      <p>javaScript</p>
+     </div>
+     <button type="button">See Project</button>
+    </article>
+   </section>
   </main>
-  <footer></footer>
+  <footer>
+   <section class="Fabout">
+    <h1>About<br>Myself</h1>
+    <p>Hello I’m a software developer! 
+      I can help you build a product , 
+      feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    <a href="">LET’S CONNECT</a>
+    <ul class="social_media">
+     <li><a href=""><img src="images/vector1.svg" alt=""></a></li>
+     <li><a href=""><img src="images/link.svg" alt=""></a></li>
+     <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
+     <li><a href=""><img src="images/twiter.svg" alt=""></a></li>
+     <li><a href=""><img src="images/vector.svg" alt=""></a></li>
+    </ul>
+    <button type="button">Get my resume</button>
+    <article class="container-lan">
+     <div class="lan-tle tags"><h2>Langguages</h2><i class="fa fa-angle-down"></i></div>
+     <div class="toggle">
+      <div class="lan"><img src="images/js.png" alt=""><p>JavaScript</p></div>
+      <div class="lan"><img src="images/html.png" alt=""><p>HTML</p></div>
+      <div class="lan"><img src="images/css.png" alt=""><p>CSS</p></div>
+     </div>
+     <div class="lan-tle"><h2>Frameworks</h2><i class="fa fa-angle-right"></i></div>
+     <div class="lan-tle"><h2>Skills</h2><i class="fa fa-angle-right"></i></div>
+    </article>
+   </section>
+   <section class="contact">
+    <div class="txt">
+     <h1>Contact me</h1>
+     <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    </div>
+    <form action="index.html">
+     <input type="text" name="name" id="name" placeholder="Yeremias">
+     <input type="email" name="email" id="email" placeholder="natanjayanj14@gmail.com">
+     <textarea name="message" id="message" cols="30" rows="10">Write your message here</textarea>
+     <input type="submit" value="Get In Touch">
+    </form>
+    <div class="line"></div>
+   </section>
```

Comment:
- Great job implementing this contact form. But considering this was not in the requirements for this project, I won't be reviewing it.
> Keep in mind that you will have a project to implement this in the future, with its own requirements.